test_dir/new_file_1

```diff
@@ -0,0 +1,8 @@
+lets see, here is the file
+well heres what
+what now
```

Comment:
RiflerRick committed on a restricted repository: BB_MCS_Test belonging to Rajdeep Mukherjee(RiflerRick) in the file test_dir/new_file_1